fall18/p2/test.py

```diff
@@ -0,0 +1,174 @@
+#!/usr/bin/python
+import subprocess, json, re, sys
+from inspect import getmembers, isfunction
+import main
+
+PASS = 'PASS'
+EPSILON = 0.0001
+
+TESTS_FUNC = [
+    ("getMaximumLand", ["Arizona", "California", "Connecticut"], 163694.74), #1
+    ("getMinimumPopulationDensity", ["Wisconsin", "Iowa", "Minnesota", 2010], 54.13546968775862), #2
+    ("predictPopulation", ["Hawaii", 2000, 2010, 0.4], 66147678.89670547), #3
+    ("predictPopulation", ["Hawaii", 2000, 2010], 3293299.011605786), #4
+    ("calcGrowthRate", ["Hawaii", 2000, 2015], 0.010596535979501064) #5
+]
+
+TESTS_PRINT = [
+    ["The area of Wisconsin is ", 65496.38], #1
```

Comment:
Why is both this number 1 and an early test is also number 1?

Reply:
I'll update.